src/modules/endpoint/DatabaseConnector.js

```diff
@@ -373,7 +373,7 @@ class DatabaseConnector {
         code: jsonObject.code,
         live: jsonObject.live,
         eventsEnabled: jsonObject.events_enabled,
-        extrasEnabled: jsonObject.extras_enabled,
+        offersEnabled: jsonObject.extras_enabled,
```

Comment:
This has to stay for backwards compatibility, right?

Reply:
Hmm. I am not very familiar with the database files in the app, but I would geuess this belongs to [NATIVE-549](https://issues.integreat-app.de/browse/NATIVE-549)